app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Do not inherit from `BaseException` it a base exception for errors that tells to the interpreter that it should exit (GeneratorExit, KeyboardInterrupt, SystemExit inherit from it). Use just `Exception`